<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="cyclone1" name="org.wso2.carbon.identity.application.authentication.framework.inbound.HttpIdentityRequestFactoryTest" tests="9" failures="0" timestamp="4 Jun 2018 12:30:16 GMT" time="2.006" errors="0">
  <testcase name="testInit" time="0.054" classname="org.wso2.carbon.identity.application.authentication.framework.inbound.HttpIdentityRequestFactoryTest"/>
  <testcase name="testInitWithDuplicateEventListenerProperties" time="0.654" classname="org.wso2.carbon.identity.application.authentication.framework.inbound.HttpIdentityRequestFactoryTest"/>
  <testcase name="testCreate" time="0.008" classname="org.wso2.carbon.identity.application.authentication.framework.inbound.HttpIdentityRequestFactoryTest"/>
  <testcase name="testInitWithDuplicateEventListenerProperties" time="1.168" classname="org.wso2.carbon.identity.application.authentication.framework.inbound.HttpIdentityRequestFactoryTest"/>
  <testcase name="testHandleRuntimeException" time="0.002" classname="org.wso2.carbon.identity.application.authentication.framework.inbound.HttpIdentityRequestFactoryTest"/>
  <testcase name="testCanHandle" time="0.005" classname="org.wso2.carbon.identity.application.authentication.framework.inbound.HttpIdentityRequestFactoryTest"/>
  <testcase name="testInitWithDuplicateEventListenerProperties" time="0.003" classname="org.wso2.carbon.identity.application.authentication.framework.inbound.HttpIdentityRequestFactoryTest"/>
  <testcase name="testCreate" time="0.105" classname="org.wso2.carbon.identity.application.authentication.framework.inbound.HttpIdentityRequestFactoryTest"/>
  <testcase name="testHandleFrameworkClientException" time="0.007" classname="org.wso2.carbon.identity.application.authentication.framework.inbound.HttpIdentityRequestFactoryTest"/>
</testsuite> <!-- org.wso2.carbon.identity.application.authentication.framework.inbound.HttpIdentityRequestFactoryTest -->
